.github/workflows/lib-injection.yml

```diff
@@ -146,6 +146,7 @@ jobs:
       DOCKER_IMAGE_WEBLOG_TAG: ${{ github.sha }}-${{ matrix.runtime }}
       RUNTIME: ${{ matrix.runtime }}
       BUILDX_PLATFORMS: linux/amd64
+      MODE: manual
```

Comment:
Do you have more details on what it fixes and how?

Reply:
I have just added the details in the summary